multi-flashcards/AsyncContentLoader/AsyncContentView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  AsyncContentView.swift
+//  Snowport
+//
+//  Created by Peyton McKee on 1/4/24.
+//
+
+import SwiftUI
+
+struct AsyncContentView<Source: LoadableObject, Content: View>: View {
+    @ObservedObject var source: Source
+    var content: (Source.Output) -> Content
+        
+    var body: some View {
+        switch source.state {
+        case .idle:
+            Color.clear
```

Comment:
This feels like an anti-pattern ...
Could you wrap the switch in a Group and put a task on the group?

Reply:
asked on slack